log/logging.go

```diff
@@ -135,19 +134,14 @@ func SetPrefix(prefix string) {
 // Fatal outputs a severe error message just before terminating the process.
 // Use judiciously.
 func Fatal(id, description string, keysAndValues ...interface{}) {
-	if Level < LevelFatal {
-		return
-	}
-	DefaultLogger.logMessage(LevelFatalName, id, description, keysAndValues...)
-	osExit(1)
+	keysAndValues = append([]interface{}{"id", id}, keysAndValues...)
```

Comment:
This might be weird for any existing calls to this function that already includes an "id" key and value. You'd essentially get two.

Maybe an alternate solution would be to prefix the description, so the output is backwards compatible:

`description = fmt.Sprintf("%s | %s", id, description)`